plugins/telepathy/telepathy.pro

```diff
@@ -30,7 +30,9 @@ CONFIG += c++11
 PKGCONFIG += Qt5Contacts
 PKGCONFIG += TelepathyQt5 qtcontacts-sqlite-qt5-extensions
 
-system(qdbusxml2cpp -c BuddyManagementAdaptor -a buddymanagementadaptor.h:buddymanagementadaptor.cpp com.nokia.contacts.buddymanagement.xml)
+system(qdbusxml2cpp -c BuddyManagementAdaptor -a buddymanagementadaptor com.nokia.contacts.buddymanagement.xml)
+system(qdbusxml2cpp -c DevicePresenceAdaptor -a devicepresenceadaptor org.nemomobile.DevicePresenceIf.xml)
+system(qdbusxml2cpp -c DevicePresenceRequestAdaptor -a devicepresencerequestadaptor org.nemomobile.DevicePresenceRequestIf.xml)
```

Comment:
There is a way to do this without resorting to system().
QDBUSXML2CPP_ADAPTOR_HEADER_FLAGS += ""
QDBUSXML2CPP_ADAPTOR_SOURCE_FLAGS += ""
QDBUSXML2CPP_INTERFACE_HEADER_FLAGS += ""
QDBUSXML2CPP_INTERFACE_SOURCE_FLAGS += ""